<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>geometryWidget</class>
 <widget class="QWidget" name="geometryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>207</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="geoFileLocationLabel">
     <property name="text">
      <string>Geometry file location:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="geoNameLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLineEdit" name="fileLocation"/>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="findGeoFileButton">
     <property name="text">
      <string>Find...</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QListWidget" name="geoNameList"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
